main.nf

```diff
@@ -44,6 +44,8 @@ def helpMessage() {
       --saveReference               Saves reference genome indices for later reusage
 
     Skipping                        Skip any of the mentioned steps
+      --skip_collapse               Skip merging Forward and Reverse reads together. (Only for pairedEnd samples)
```

Comment:
I think it would be clearer/more consistent to have a `--skip_adapterremoval` flag here (which switches both `--skip_collapse` and `--skip_trim` to false).

Then we move the `--skip_collapse` and `--skip_trim` in help message to 'AdapterRemoval' section, as   in that section you're actually modifying the _module_, not turning entire modules on and off.